### PR TITLE
Update EIP-1: Add IETF as a permissible origin

### PR DESCRIPTION
**Pull request by SamWilsn**
Originally from https://github.com/ethereum/EIPs/pull/6691 by @kdenhartog.

Splitting out so it can be discussed separately.

Changed slightly with my review feedback.

### CODE REVIEW

**Review by kdenhartog — DISMISSED** [review]
LGTM thanks for following up on this since I never got back to it. I'll convert mine to focus just on WHATWG since I'm going to what to reference it in EIP-6963

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -252,6 +252,24 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Internet Engineering Task Force (IETF)
```
Comment (Pandapip1) [inline EIPS/eip-1.md:255]: Again, would mildly prefer thr following:


```suggestion
### IETF Request for Comments (RFC)
```

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -252,6 +252,24 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
...
+
+Links to a Request For Comment (RFC) specification may be included using normal markdown syntax, such as:
```
Comment (Pandapip1) [inline EIPS/eip-1.md:257]:
```suggestion
Links to an IETF Request For Comment (RFC) specification may be included using normal markdown syntax, such as:
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Again, I think we should ensure that EIP authors use the right reference (`RFC X`)

**Pull request merged**